app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -163,6 +164,10 @@ register_katello_host | bash
 register_host | bash
 <% end -%>
 
+<% if plugin_present?('katello') && truthy?(@setup_container_registry_certs) -%>
+<%= snippet('container_certs_setup') -%>
+container_certs_setup
+<% end -%>
```

Comment:
This is rendering weird:

```
  # Locate and link client certificate
  cert_file=$(find /etc/pki/entitlement/ -maxdepth 1 -name '*.pem' -type f | grep -v -- '-key.pem' | head -n1)
  if [ -z "$cert_file" ]; then
    echo "Error: Client certificate file not found in /etc/pki/entitlement/"
    exit 1
  fi
  ln -sf "$cert_file" "$cert_dir/client.cert"
}container_certs_setup
echo 'Hey it works'
cleanup_and_exit
```

`echo 'Hey it works'` is my after_registration snippet.

Reply:
Seems to render fine for me as is.. 😕 
```
....
  fi
  ln -sf "$cert_file" "$cert_dir/client.cert"
}
container_certs_setup
echo "Hello!"
cleanup_and_exit
```

Reply:
Working well now!